V2.0.0 Updates:

intervalia opened the pull request:
**This release includes breaking changes.**

● Removed the escaping of the back-tick in templates. This was preventing sub-ES6 Template Literals in the templates.
● You now must list the source folders. In most cases you would change from srcPath: "modules/src" to srcPath: "modules/src/*"
● Changed default build types from MJS and IIFE to MJS and CJS since these can both be loaded in a similar manner.
● addKELocale is now false by default
● alwaysReturnFile is now false by default.
● defaultLocaleVariable is now set to document.documentElement.lang which is the value set in the lang attribute of the <html> tag: <html lang="fr"> would use fr as the default value when getting the lang object.
● includePath is now false by default.
● New config options dstExtCJS, dstExtCJS5, dstExtIIFE, dstExtIIFE5 and dstExtMJS allow you to set the output extension for the various output file types.
● Added ability for distPath to be an object and not just a string.
● Added and cleaned up Docs
● Added more testing for the new code.

intervalia committed:
V2.0.0 Updates:
● Removed the escaping of the back-tick in tempaltes. This was preventing sub-ES6 Template Literals in the templates.
● You now must list the source folders. In most cases you would change from srcPath: "modules/src" to srcPath: "modules/src/*"
● Changed default build types from MJS and IIFE to MJS and CJS since these can both be loaded in a similar manner.
● addKELocale is now false by default
● alwaysReturnFile is now false by default.
● defaultLocaleVariable is now set to document.documentElement.lang which is the value set in the lang attribute of the <html> tag: <html lang="fr"> would use fr as the default value when getting the lang object.
● includePath is now false by default.
● New config options dstExtCJS, dstExtCJS5, dstExtIIFE, dstExtIIFE5 and dstExtMJS allow you to set the output extension for the various output file types

GyozaGuy commented on README.md:
```diff
@@ -208,6 +289,36 @@ templates.str = function(key, data) {
 
+### Passing data into a template
+
+The signature of the functions `tempates.str` and `tempaltes.dom` is:
```
`tempates.str` --> `templates.str`
`tempaltes.dom` --> `templates.dom`

intervalia replied:
picky, picky!! :)

GyozaGuy commented on README.md:
````diff
@@ -208,6 +289,36 @@ templates.str = function(key, data) {
+function(key, data) {}
+```
+
+So when your code calls either `tempates.str` and `tempaltes.dom` you can pass any object as the second parameter.
````
`tempates.str` --> `templates.str`
`tempaltes.dom` --> `templates.dom`

GyozaGuy commented on README.md:
```diff
@@ -208,6 +289,36 @@ templates.str = function(key, data) {
+
+> The first parameter is they key used to get the correct template.
+
+If you wanted to create a table with an number or rows defined by a variable you would create your template like this:
```
`an number or rows` --> `a number of rows`

GyozaGuy commented on README.md:
```diff
@@ -239,10 +350,9 @@ The import line of code will be inserted into the top of `templates.mjs`. _You c
 
 | Date | Version | Description |
 | --- | --- | --- |
+| 06/05/2018 | 2.0.0 | **Breaking Changes!!**<br>&#x25cf; Removed the escaping of the back-tick in tempaltes. This was preventing sub-ES6 Template Literals in the templates.<br/>&#x25cf; You now must list the source folders. In most cases you would change from `srcPath: "modules/src"` to `srcPath: "modules/src/*"`<br/>&#x25cf; Changed default build types from MJS and IIFE to MJS and CJS since the ...
```
`tempaltes` --> `templates`

GyozaGuy commented on rollup.root.config.js:
```diff
@@ -20,17 +21,23 @@ function init(theirConfig = {}) {
-    buildTypes: [ BUILD_TYPES.MJS, BUILD_TYPES.IIFE ],
+    addKELocale: false, // Add the KE locale if it does not exist
+    alwaysReturnFile: false,
+    buildTypes: [ BUILD_TYPES.MJS, BUILD_TYPES.CJS ],
```
`IIFE` doesn't belong in buildTypes anymore?

intervalia replied:
This is the DEFAULT set of builds. We have always overridden them.

GyozaGuy reviewed (changes requested):
Some minor spelling issues in the readme

intervalia committed:
Spelling corrections

intervalia committed:
More changes for v2.0.0
● Added ability for distPath to be an object and not just a string.
● Added and cleaned up Docs
● Added more testing for the new code.
